src/components/header_menu.jsx

```diff
@@ -1,11 +1,81 @@
-import React from 'react'
-
-const HeaderMenu = props => (
-  <header className={'page-header-menu ' + (props.className || '')}>
-    <div className="page-header-menu--left">{props.left}</div>
-    {props.children}
-    <div className="page-header-menu--right">{props.right}</div>
-  </header>
-)
+import React, { useMemo } from 'react'
+
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Link from 'cozy-ui/transpiled/react/Link'
+import { SharedRecipients } from 'cozy-sharing'
+import { Typography } from '@material-ui/core'
+import { Slugs } from 'constants/strings'
+import { NotePath } from './notes/List/NotePath'
+import { CozyFile } from 'cozy-doctypes'
+import { getDriveLink } from 'lib/utils'
+
+import styles from './header_menu.styl'
+import { WithBreakpoints } from './notes/List/WithBreakpoints'
+import { Breakpoints } from 'types/enums'
```

Comment:
Please respect import orders: 
1/ third parties
2/ cozy's lib 
3/ internal import

Reply:
Oversight from my end, usually I respect the order. Will update